examples/adafruitio_27_wifimanager/adafruit_27_wifimanager.ino

```diff
@@ -0,0 +1,83 @@
+/* Adafruit IO Example Using WiFiManager
+ *
+ * This is a simple Adafruit feed subscribe example that uses
+ * WiFiManager to handle setup of WiFi credentials and connecting
+ * to the network instead of defining the WiFI SSID and password
+ * explicitly in the code.
+ *
+ * To use this example, add your Adafruit IO Username and Key
+ * and setup a feed called "myfeed".  When you manually add data
+ * to the feed on io.adafruit.com, you'll see that data written to
+ * the serial output.
+ *
+ * Brad Black - 2022
+ *
+ */
+
+#include <WiFiManager.h>
+#include "AdafruitIO_WiFi.h"
+
+char IO_USERNAME[64] = "your username";
+char IO_KEY[64] = "your key";
+AdafruitIO_WiFi io(IO_USERNAME, IO_KEY);
+AdafruitIO_Feed *myfeed = io.feed("myfeed");
+
+WiFiManager wifiManager;
+
+void handleMessage(AdafruitIO_Data *data)
+{
+
+  Serial.print("received <- ");
+  Serial.println(data->toString());
+
+} // handleMessage
+
+void setup()
+
+{
+  WiFi.mode(WIFI_STA);
+  Serial.begin(57600); // Initialize serial port for debugging.
```

Comment:
Please use the library default of `115200` baud

Reply:
Done.